src/components/manage/Blocks/Search/components/DateRangeFacet.jsx

```diff
@@ -0,0 +1,118 @@
+import React, { useState } from 'react';
+import { Header } from 'semantic-ui-react';
+import { defineMessages, injectIntl } from 'react-intl';
+import { injectLazyLibs } from '@plone/volto/helpers/Loadable/Loadable';
+import { compose } from 'redux';
+import { Icon } from '@plone/volto/components';
+
+import leftKey from '@plone/volto/icons/left-key.svg';
+import rightKey from '@plone/volto/icons/right-key.svg';
+import clearSVG from '@plone/volto/icons/clear.svg';
+
+import 'react-dates/initialize';
+import 'react-dates/lib/css/_datepicker.css';
+
+const messages = defineMessages({
+  startDate: {
+    id: 'Start Date',
+    defaultMessage: 'Start Date',
+  },
+  endDate: {
+    id: 'End Date',
+    defaultMessage: 'End Date',
+  },
+});
+
+const PrevIcon = () => (
+  <div
```

Comment:
Just it case somebody wants to style these controls, it will be easier for them if this style is assigned to a class.

Reply:
Good call, I'll update it.